images/libvirt-kubevirt/Dockerfile

```diff
@@ -44,4 +44,12 @@ RUN rm -f /libvirtd.sh
 COPY libvirtd.sh /libvirtd.sh
 RUN chmod a+x /libvirtd.sh
 
+RUN echo "unix_sock_group=\"libvirt\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "unix_sock_ro_perms=\"0777\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "unix_sock_rw_perms=\"0770\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "auth_unix_ro=\"none\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "auth_unix_rw=\"none\"" >> /etc/libvirt/libvirtd.conf
```

Comment:
@cynepco3hahue We can use Kubernetes secrets (https://kubernetes.io/docs/concepts/configuration/secret/) to provide the credentials. A container can find the decrypted secrets in a special file, which is mounted by Kubernetes. To use the credentials, we have to modify our start-up scripts for libvirt and virt-handler. The libvirt start-script has to take the secrets from the file and place them in the server config, for virt-handler we have to place them in the client config. By doing this on every container start, we don't have to hardcode them.